src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Client/wwwroot/index.html

```diff
@@ -8,16 +8,17 @@
     <base href="/" />
     <link href="css/bootstrap/bootstrap.min.css" rel="stylesheet" />
     <link href="css/app.css" rel="stylesheet" />
-<!--#if (Hosted)  -->
+    <!--#if (Hosted)  -->
     <link href="ComponentsWebAssembly-CSharp.Client.styles.css" rel="stylesheet" />
-<!--#else  -->
+    <!--#else  -->
     <link href="ComponentsWebAssembly-CSharp.styles.css" rel="stylesheet" />
-<!--#endif  -->
+    <!--#endif  -->
```

Comment:
Could we revert these changes, I believe the editor may've just done these on it's own.